examples/ee-demo/app.js

```diff
@@ -39,64 +37,44 @@ export default class App extends React.Component {
 
     this.eeApi = new EEApi();
 
-    this.state = {
-      viewState: defaultViewState,
-      eeObject: null,
-      visParams: null
-    };
+    this.state = {loggedIn: false};
   }
 
   async _onLoginSuccess(user, loginProvider) {
     // TODO - called twice, this should not be needed
-    if (this.loggedIn) {
+    const {loggedIn} = this.state;
+    if (loggedIn) {
       return;
     }
-    this.loggedIn = true;
 
     this.forceUpdate();
     // Client id to your EE application
     await this.eeApi.initialize({clientId: EE_CLIENT_ID});
-
-    // Old elevation example
-    // Eventually we'll have multiple examples
-    // const eeObject = ee.Image('CGIAR/SRTM90_V4').serialize();
-    // this.setState({eeObject});
-    const eeObject = ee
-      .ImageCollection('NOAA/GFS0P25')
-      .filterDate('2018-12-22', '2018-12-23')
-      .limit(48)
-      .select('temperature_2m_above_ground');
-    const visParams = {
-      min: -40.0,
-      max: 35.0,
-      palette: ['blue', 'purple', 'cyan', 'green', 'yellow', 'red']
-    };
-    this.setState({eeObject, visParams});
-  }
-
-  _onViewStateChange({viewState}) {
-    this.setState({viewState});
+    // Must be after initialization
+    this.setState({loggedIn: true});
   }
 
   render() {
-    const {eeObject, visParams, viewState} = this.state;
-
-    const layers = eeObject && [
-      new EarthEngineLayer({
-        eeObject,
-        visParams,
-        opacity: 0.5
-      })
-    ];
+    const {exampleName = 'imagecollection'} = this.props;
+    const {loggedIn} = this.state;
+
+    let layers;
+    if (loggedIn) {
+      switch (exampleName.toLowerCase()) {
+        case 'image':
+          layers = ImageExampleLayers();
```

Comment:
CapitalCase is usually reserved for class names. How about `renderImageExampleLayers()`